generative_models/gaussian_discriminant_analysis.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import pandas as pd
```

Comment:
It seems that you did not use pandas package in your code. So I think you could remove this line.